packages/block-library/src/navigation-link/shared/test/controls.js

```diff
@@ -95,18 +95,22 @@ describe( 'Controls', () => {
 		expect( urlInput.value ).toBe( 'https://example.com/test page' );
 	} );
 
-	it( 'encodes URL values when changed', () => {
+	it( 'calls updateAttributes with new URL on blur', () => {
 		render( <Controls { ...defaultProps } /> );
 
 		const urlInput = screen.getByLabelText( 'Link' );
 
+		fireEvent.focus( urlInput );
 		fireEvent.change( urlInput, {
 			target: { value: 'https://example.com/test page' },
 		} );
+		fireEvent.blur( urlInput );
```

Comment:
Are we able to do this via interactions with elements rather than programmatically? If not then why? I'm curious about why we do it via APIs here.

Reply:
Nope - updated to userEvents 👍🏻